packages/web-components/fast-components/src/switch/switch.styles.ts

```diff
@@ -57,214 +53,169 @@ export const switchStyles: (
              * Chromium likes to select label text or the default slot when
              * the checkbox is clicked. Maybe there is a better solution here?
              */ ""
-        } user-select: none;
+        }
+        user-select: none;
+        width: max-content;
     }
-
-    :host([disabled]) {
+    :host(.disabled) {
```

Comment:
Maybe I missed something, are we making the change from attributes to class names everywhere? I think attributes are closer to the platform, but I know they don't work in all cases. Is this our standard?